core/src/main/scala/kafka/server/KafkaRequestHandler.scala

```diff
@@ -129,6 +129,7 @@ class BrokerTopicMetrics(name: Option[String]) extends KafkaMetricsGroup {
   val failedFetchRequestRate = newMeter(BrokerTopicStats.FailedFetchRequestsPerSec, "requests", TimeUnit.SECONDS, tags)
   val totalProduceRequestRate = newMeter(BrokerTopicStats.TotalProduceRequestsPerSec, "requests", TimeUnit.SECONDS, tags)
   val totalFetchRequestRate = newMeter(BrokerTopicStats.TotalFetchRequestsPerSec, "requests", TimeUnit.SECONDS, tags)
+  val intraBrokerBytesRate = newMeter(BrokerTopicStats.IntraBrokerBytesPerSec, "bytes", TimeUnit.SECONDS, tags)
```

Comment:
Yeah it wasn't in the KIP.. It seems useful and reasonable to include in this patch. I can document this sensor in the KIP-113 wiki.

I think it may be useful to have it as topic level for debug purpose. But I am not strong on this. I have updated the patch to only record it for `allTopicsStats`.

Reply:
If we enable replication quota, the quota metric already tracks the byte rate across all moving partitions. So this seems redundant?

Reply:
You are right. We no longer need this metric now that we will apply quota to all topic partitions.